src/lib.rs

```diff
@@ -630,55 +676,120 @@ impl<'arg, A: Argument + 'arg, I: Iterator<Item = A>> Options<A, I> {
     /// let args = ["--flag", "one", "two"];
     /// let mut opts = Options::new(args.into_iter());
     ///
-    /// assert_eq!(opts.next(), Ok(Some(Opt::Long("flag"))));
-    /// assert_eq!(opts.next(), Ok(None));
+    /// assert_eq!(opts.next_opt(), Ok(Some(Opt::Long("flag"))));
+    /// assert_eq!(opts.next_opt(), Ok(None));
     ///
-    /// let mut args = opts.args();
+    /// let mut args = opts.positionals();
     ///
     /// assert_eq!(args.next(), Some("one"));
     /// assert_eq!(args.next(), Some("two"));
     /// assert_eq!(args.next(), None);
     /// ```
-    pub fn args(&mut self) -> ArgIterator<A, I> {
-        ArgIterator::new(self)
+    pub fn positionals(&mut self) -> Positionals<A, I> {
+        Positionals::new(self)
+    }
+
+    /// Consumes this [`Options`], returning an iterator over the rest
+    /// of the arguments. The returned iterator wraps the one originally
+    /// passed to [`Options::new`].
+    ///
+    /// # Panics
+    ///
+    /// Panics if an option is currently being parsed.
+    pub fn into_positionals(self) -> IntoPositionals<A, I> {
+        match self.state {
+            State::Start { .. } | State::EndOfOption(_) | State::End { .. } => {
+                IntoPositionals::new(None, self.iter)
+            }
+            State::Positional(positional) => IntoPositionals::new(Some(positional), self.iter),
+            _ => {
+                panic!("called Options::into_positionals() while option parsing hasn't finished")
+            }
+        }
+    }
+
+    /// Retrieves the next *argument*. An *argument* is represented by
+    /// [`Arg`] and defined as either an option ([`Opt`]) or a
+    /// positional argument.
+    ///
+    /// This method is implemented entirely using other public methods
+    /// of [`Options`].
+    ///
+    /// # Panics
+    ///
+    /// Panics if [`Options::next_opt`] does.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use getargs::{Arg, Opt, Options};
+    /// #
+    /// let args = ["--option", "VALUE", "arg", "arg2", "-vvv", "--option", "VALUE", "--", "arg3", "arg4", "--not-an-option"];
+    /// let mut opts = Options::new(args.into_iter());
+    ///
+    /// // Regular options and values work...
+    /// assert_eq!(opts.next_arg(), Ok(Some(Arg::Long("option"))));
+    /// assert_eq!(opts.value(), Ok("VALUE"));
+    ///
+    /// // Positional arguments after work...
```

Comment:
Typo, should be "also work"

Reply:
I didn't mean "also", I meant (Positional arguments after) work, perhaps it should be "Positional arguments afterwards also work"